newIDE/app/src/BehaviorsEditor/index.js

```diff
@@ -344,6 +507,18 @@ const BehaviorsEditor = (props: Props) => {
           </ScrollView>
           <Column>
             <Line justifyContent="flex-end" expand>
+              {isClipboardContainingBehaviors && (
+                <Column>
+                  <FlatButton
+                    key={'paste-behaviors'}
+                    leftIcon={<PasteIcon />}
+                    label={<Trans>Paste</Trans>}
+                    onClick={() => {
+                      pasteBehaviors();
+                    }}
```

Comment:
Would this work?
```suggestion
                    onClick={pasteBehaviors}
```

Reply:
`pasteBehaviors` is async because it can ask a question.

Reply:
And flow is not happy?

Reply:
Yes, because of the Promise that is returned.